rpcserver.go

```diff
@@ -915,10 +920,10 @@ func (r *rpcServer) DecodeAddr(_ context.Context,
 	return rpcAddr, nil
 }
 
-// VerifyProof attempts to verify a given proof file that claims to be anchored
-// at the specified genesis point.
+// VerifyProof attempts to verify a given proof file that claims to be
+// anchored at the specified genesis point.
```

Comment:
This doesn't need to be changed, looks like the comment fit the 80 characters before.